### ISSUE TITLE
Request: add host to "no route to host" error message

### ISSUE THREAD

**Issue by onnozweers** [dCache/dcache#6954]
Dear dCache devs,

I see a lot of these in the log of a WebDAV door:

17 Jan 2023 14:51:16 (webdav2883-penguin14) [door:webdav2883-penguin14@webdav2883-penguin14Domain:AAXydf4qxug] Internal server error: org.dcache.webdav.WebDavException: Problem with transferred data: No route to host (Host unreachable)

The WebDAV door is on host penguin14. But which host is it trying to contact? We have ~250 nodes in our dCache cluster.

Could the host that is being contacted, be added to the error message? That would make troubleshooting a lot easier.

Kind regards,
Onno

**Comment by paulmillar** [comment]
I agree with you: the error message should include the hostname/ip-address in the error message.

If it helps, this is (likely) a proxied upload (HTTP PUT) request where the door cannot contact the pool node.

**Comment by onnozweers** [comment]
Hi Paul, thanks! That's what I thought. Only a percentage of these uploads fails. Such issues are usually more difficult to troubleshoot. In such cases, analysing logs with `grep / awk / sed | sort | uniq --count` can be very helpful to find patterns.

It would be great if the contacted port could be included too, but I'm not sure if that is easy to accomplish. That may help to find patterns in port ranges (possible firewall issues).
